### DISCUSSION TITLE
Expandir EmptyMessage para ofrecer dos alternativas de peso similar

### DISCUSSION THREAD

**Discussion by fonse** [TiendaNube/nimbus-design-system#201] [Q&A] [answered]
Estamos revisando temas de onboarding y queremos modificar la empty screen de productos para ofrecer más opciones:

![image](https://github.com/TiendaNube/nimbus-design-system/assets/605137/efeaeca6-90aa-4ac0-a01e-81c37e213cf5)

Con el componente [EmptyMessage](https://nimbus.tiendanube.com/es-AR/documentation/patterns/empty-message) puedo armar la parte de arriba de los call to actions, pero necesitaría poder sumarle la mitad de abajo para un cal to action alternativo de peso similar al principal.

**Comment by juanchigallego** [comment, marked as answer]
Hola @fonse ! 
El componente actualmente no tiene la posibilidad de agregar más contenido dentro, pero ya que la prop `actions` acepta un ReactNode, es posible incluir ese contenido adicional en esa prop, como workaround.
En caso de aparecer otro caso donde se necesite agregar contenido adicional en un `EmptyMessage`, podemos ver de modificar el componente agregando una nueva prop.

Cualquier duda avisanos!
